cellprofiler/modules/plugins/measurementtemplate.py

```diff
@@ -1,59 +1,147 @@
+import cellprofiler.measurement
+import cellprofiler.object
+import cellprofiler.setting
 
-from centrosome.zernike import construct_zernike_polynomials
-from centrosome.zernike import get_zernike_indexes
-from centrosome.cpmorphology import minimum_enclosing_circle
-from centrosome.cpmorphology import fixup_scipy_ndimage_result as fix
+import centrosome.zernike
+import centrosome.cpmorphology
 
-##################################
+
+__doc__ = """\
+MeasurementTemplate
+===================
+
+**MeasurementTemplate** - an example measurement module. It's recommended to
+put a brief description of this module here and go into more detail below.
+
+This is an example of a module that measures a property of an image both
+for the image as a whole and for every object in the image. It
+demonstrates how to load an image, how to load an object and how to
+record a measurement. 
+
+The text you see here will be displayed as the help for your module, formatted
+as `reStructuredText <http://docutils.sourceforge.net/rst.html>`_.
+ 
+Note whether or not this module supports 3D image data and respects masks.
+A module which respects masks applies an image's mask and operates only on
+the unmasked data. Update the table below to indicate which image processing
+features this module supports.
+
+|
+
+============ ============ ===============
+Supports 2D? Supports 3D? Respects masks?
+============ ============ ===============
+YES          NO           YES
+============ ============ ===============
+
+See also
+^^^^^^^^
+
+Is there another **Module** that is related to this one? If so, refer
+to that **Module** in this section. Otherwise, this section can be omitted.
+
+What do I need as input?
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Are there any assumptions about input data someone using this module
+should be made aware of? For example, is there a strict requirement that
+image data be single-channel, or that the foreground is brighter than
+the background? Describe any assumptions here.
+
+This section can be omitted if there is no requirement on the input.
+
+What do I get as output?
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Describe the output of this module. This is necessary if the output is
+more complex than a single image. For example, if there is data displayed
+over the image then describe what the data represents.
+
+This section can be omitted if there is no specialized output.
+
+Measurements made by this module
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Describe the measurements made by this module, if any. Typically, measurements
+are described in the following format:
+
+**Measurement category:**
+
+-  *MeasurementName*: A brief description of the measurement.
+-  *MeasurementName*: A brief description of the measurement.
+
+**Measurement category:**
+
+-  *MeasurementName*: A brief description of the measurement.
+-  *MeasurementName*: A brief description of the measurement.
+
+This section can be omitted if no measurements are made by this module.
```

Comment:
I don't think this belongs in this section, as I assume measurement modules make measurements (perhaps that's an improper assumption).